.github/scripts/install_hwloc.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# install_hwloc.sh - Script for building and installing HWLOC library from source code
+
+set -e
+
+git clone -b hwloc-2.10.0 https://github.com/open-mpi/hwloc.git
```

Comment:
you wrote in reqs that we need 2.3 but you install 2.10?

Reply:
Well, I've tested it with 2.3.0, but 2.10.0 is the latest version available.